infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationManager.java

```diff
@@ -258,14 +257,14 @@ public void enforceSnapshotSync(DiscoveryServiceEvent event) {
     }
 
     /**
-     * Update Replication Status as NOT_STARTED.
+     * Initialize Replication Status as NOT_STARTED.
      * Should be called only once in an active lifecycle.
      */
-    public void updateStatusAsNotStarted() {
+    public void initializeStatusAsNotStarted() {
         runtimeToRemoteCluster.values().forEach(corfuLogReplicationRuntime ->
                 corfuLogReplicationRuntime
                         .getSourceManager()
                         .getAckReader()
-                        .markSyncStatus(SyncStatus.NOT_STARTED));
+                        .markInitialSyncStatus());
```

Comment:
why do we use the ack reader to initialize the status?  why not use the metadata manager directly?

Reply:
When debugging the IT failure for testNewConfigWithSwitchRole I noticed that the cluster id was the same for the both active and standby which should not have been the case. I think this stemmed from using the localClusterId of the metadata manager instead of the remoteClusterId from the ack reader which is why the ack reader is used here.